_includes/data/data_dictionary.html

```diff
@@ -3,7 +3,7 @@
     The <a href="https://bcda.cms.gov/assets/data/CCLF_BCDA_BB_Crosswalk.xlsx" target="blank">CCLF to BCDA Data Dictionary</a> is provided to BCDA users to facilitate their data mapping process.
```

Comment:
Should we update this so it's a relative path? So `/assets/data/CCLF_BCDA_BB_Crosswalk.xlsx`

Reply:
We def should be using relative links. Just changed it, but still no change in the behavior for me.